src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -147,7 +147,7 @@ protected function doSave(array $values, $lifetime)
     private function getFile($id, $mkdir = false)
     {
         $hash = str_replace('/', '-', base64_encode(md5($id, true)));
```

Comment:
MD5 weaknesses make it not suitable for any security related job (signatures, etc.). Here, we only need to be reasonably sure that keys won't collide. Making two bytes case insensitive won't invalidate this.
As far as security is concerned, *if* a collision occurs, it will be catch by the unhashed key comparison done when opening files.